### ISSUE TITLE
Find a replacement for qtranslate-x

### ISSUE THREAD

**Issue by MarcelloPerathoner** [cceh/capitularia#62]
The translation plugin qtranslate-x is no longer maintained. We need a replacement.

**Comment by MarcelloPerathoner** [comment]
https://wordpress.org/plugins/qtranslate-x/
https://github.com/qTranslate-Team/qtranslate-x
https://github.com/qTranslate-Team/qtranslate-x/issues/579

**Comment by MarcelloPerathoner** [comment]
This is a promising candidate: https://github.com/qtranslate/qtranslate-xt

**Comment by MarcelloPerathoner** [comment]
Note: testing of qtranslate-xt is live starting now.
```
cd wp-content/plugins/
wget https://github.com/qtranslate/qtranslate-xt/archive/3.6.2.zip
mv 3.6.2.zip qtranslate-xt-3.6.2.zip 
unzip qtranslate-xt-3.6.2.zip 
```
Go to my sites | network admin | dashboard | plugins, deactivate the qtranslate-x and activate the qtranslate-xt plugin.

**Comment by MarcelloPerathoner** [comment]
Manually updated to 3.8.1.
Download new version, unzip into its own version directory, goto wp admin page, disable old version, enable new version.

**Comment by MarcelloPerathoner** [comment]
Manually updated to 3.11.0.

**Comment by MarcelloPerathoner** [comment]
Manually updated to version 3.12.0

**Comment by monkey121n6** [comment]
Ja, das Plugin wird nun von der Community weiterentwickelt und ist nicht über die Wordpress-Update-Funktionalität updatebar. Stattdessen müssen die Plugin-Dateien aus dem Git-Repository runtergeladen und auf dem Server eingespielt werden. 
Das find ich aber umproblematisch, um Wordpress/Plugins zu updaten, muss ich eh auf den Server. 
Das Ersetzen des Plugins wäre auch ziemlich aufwändig. Ich vermute, dieses Issue wurde eröffnet, weil das Plugin ursprünglich gar nicht mehr maintained werden sollte, was problematisch ist! Es wrid aber aktuell weiter gepflegt und ich sehe keinen Grund, es zu ersetzen. Wenn ihr zustimmt, würde ich ein aktuelles Update einspielen und das Issue schließen.

**Comment by laMischke** [comment]
OK!
